.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,4 +40,7 @@ or AWS CLI commands to trigger something.
 - [ ] Fix linting errors
 - [ ] Make sure code coverage hasn't dropped
 - [ ] Provide verification config/commands/resources
-- [ ] Leave a comment that this is ready for review once you've finished the implementation
+- [ ] Change ready for review message below
+
+
+***Is this ready for review?:*** NO
```

Comment:
It was one before, but imho it was too easy for people to loose track of it and check it before the PR might have been ready for release. Before people left a message, now they simply change it from yes to no so its a more specific act that they need to do (which doesn't take long but makes sure they don't do it accidentally)